network/rtpdump.ksy

```diff
@@ -0,0 +1,73 @@
+meta:
+  id: rtmpdump
+  title: Rtpdump (rtptools)
+  file-extension: rtmpdump
+  license: Unlicense
+  ks-version: 0.1

```

Comment:
```suggestion
```

I'd rather leave this out, it doesn't really have any benefit, given that it doesn't limit the minimum KS version in any way. And declaring version 0.1 is just weird.